room_server/lib/room_server/checkers.ex

```diff
@@ -0,0 +1,131 @@
+defmodule RoomServer.Checkers do
+
+    def new_game(name) do
+
```

Comment:
Why did you choose "rc" as the representation? Would `{row, col}` not have been easier?

No deduction. Just a'wonderin'